tests/models/anthropic/conftest.py

```diff
@@ -0,0 +1,115 @@
+"""Shared fixtures for Anthropic model tests."""
+
+from __future__ import annotations as _annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from ...conftest import try_import
+from ..test_anthropic import MockAnthropic, completion_message
+
+with try_import() as imports_successful:
+    from anthropic import AsyncAnthropic
+    from anthropic.types.beta import BetaMessage, BetaTextBlock, BetaToolUseBlock, BetaUsage
+    from pydantic import BaseModel
+
+    from pydantic_ai.models.anthropic import AnthropicModel
+    from pydantic_ai.providers.anthropic import AnthropicProvider
+
+if TYPE_CHECKING:
+    from pydantic_ai import Agent
+
+
+# Model fixtures for live API tests
+@pytest.fixture
+def anthropic_sonnet_4_5(anthropic_api_key: str) -> AnthropicModel:
```

Comment:
It happens so often that we want to add model settings that I would make that an argument here as well. But then having that 3x seems repetitive. I agree repeating the 	`provider=AnthropicProvider(api_key=anthropic_api_key)` bit in tests is annoying. What if we have just `anthroic_model(model_name, model_settings)`?